python/ray/data/dataset.py

```diff
@@ -1913,7 +1926,7 @@ def write_numpy(
         self,
         path: str,
         *,
-        column: str = "value",
+        column: str = TENSOR_COL_NAME,
```

Comment:
How about renaming this enum to `DEFAULT_COL_NAME`? As mentioned in a separate comment, I don't think this should be tensor specific, and we shouldn't have logic that checks specifically against any particular column name.

Reply:
Let's centralize discussion of that on the other thread, but given that the single-column table representation of pure tensor data is a Datasets implementation detail, I think that this should be tensor-specific.